### PR TITLE
Topics

### PR DESCRIPTION
**Pull request by mhebing**
Updates based on the review of WP3 (m·stats)

### CODE REVIEW

File: static/src/js/topic_list.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /***
- * Topic List Design 1
+ * Topic List Design 2
```
Comment (afuetterer) [inline static/src/js/topic_list.js]: There is no more design 1 and 2 in this commit. Call it Topic List and remove the reference to design 1

File: static/src/js/topic_list.js [hunk 2]
```diff
@@ -70,140 +74,81 @@ $(function () {
...
         renderNode: function (event, data) {
             var node = data.node;
             var d = node.data.description || '';
             var $spanTitle = $(node.span).find('span.fancytree-title');
             if ($(node.span).find('span.filter-options').length == 0) {
-                if (node.type == 'topic' || node.type == 'concept') {
-                    $spanTitle.after(filter_options_string);
+                if (node.type == 'topic') {
+                    $spanTitle.after(filter_and_clipboard); // insert additional copy_to_clipboard button to button group
+                    // $spanTitle.before(node.key)
```
Comment (afuetterer) [inline static/src/js/topic_list.js]: Remove.

File: static/src/js/topic_list.js [hunk 3]
```diff
@@ -70,140 +74,81 @@ $(function () {
...
                 }
-                else if (node.type == 'variable' || node.type == 'question') {
-                    $spanTitle.after(filter_options_var_quest_string);
+                if (node.type == 'concept') {
+                    $spanTitle.after(filter_options_string);
+                    // $spanTitle.before(node.key)
```
Comment (afuetterer) [inline static/src/js/topic_list.js]: Remove.

**Comment by afuetterer** [comment]
Also please remove the unused for loop:
https://github.com/ddionrails/ddionrails/blob/77f1e27d07b36150db6ae693387341b130410cff/static/src/js/topic_list.js#L161

**Comment by afuetterer** [comment]
Remove this block if it is not used anymore.
https://github.com/ddionrails/ddionrails/blob/77f1e27d07b36150db6ae693387341b130410cff/templates/studies/study_topics.html#L81

**Comment by afuetterer** [comment]
Can be reviewed here:
https://ddionrails.soep.de/soep-core/topics/en

**Comment by mhebing** [comment]
@afuetterer I just pushed an update with the clean up

**Pull request merged**